src/schemata/definitions/step/table.yml

```
@@ -166,6 +166,11 @@ properties:
         - cells
     description: |
       Configures which table elements can be selected to trigger behaviors.
+  transpose:
+    type: boolean
+    default: false
+    description: |
+      Display table in a transposed position, flipping the position of rows and columns.
```

```suggestion
      Display table in a transposed position, flipping the position of rows and columns. This is especially useful for tables with many columns, but only a single row.
```

This is what we intend it for, right? Speaking for myself, without the added explanation I don't think I would have understood why and where I would ever want to use ths feature 🤔

---

pretty much, could also be two, three rows.. but, yes, generally it's for when there're sufficiently larger number of columns, but not much rows..